### PR TITLE
Feat/api mock tests

### PR DESCRIPTION
**Pull request by Nullably**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**Optional**:
- Related Github issue:
- Related Clubhouse Story:6494

**A description of the changes proposed in the pull request**:
- Create a local host web server to handle (mock) api requests
- aiohttp requests are redirected to local by mocking URL of yarl 
- Liquid market test cases have been updated to either be tested by api mocking or without
- MOCK_API_ENABLED environment variable needs to be created to configure the mock

### CODE REVIEW

File: test/integration/humming_web_app.py [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+    def start(self):
+        if self.started:
+            self.stop()
+        thread = Thread(target=self._start_web_app)
+        thread.daemon = True
+        thread.start()
+
+    def stop(self):
+        asyncio.ensure_future(self._stop())
```
Comment (martinkou) [inline test/integration/humming_web_app.py]: `ensure_future()` is no longer sufficient here considering the event loop is now in another thread.

Instead, you should use
```
asyncio.run_coroutine_threadsafe(sel.f_stop(), self._ev_loop)
```

File: test/integration/humming_web_app.py [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+            await asyncio.sleep(0.1)
+
+    async def _stop(self):
+        if self._runner is None:
+            return
+        await self._runner.cleanup()
+        self._runner = None
+        self._impl = None
+        self._port = None
+        self._started = False
```
Comment (martinkou) [inline test/integration/humming_web_app.py]: You should also call `self._ev_loop.stop()` here, to stop the event loop and end the wrapping thread.

File: test/integration/humming_web_app.py (hunk 1 above)
Comment (Nullably) [reply test/integration/humming_web_app.py]: Done.

File: test/integration/humming_web_app.py (hunk 2 above)
Comment (Nullably) [reply test/integration/humming_web_app.py]: Done.

**Pull request merged**